app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -51,6 +51,12 @@ class PlayerActivity : AppCompatActivity() {
      */
     private val hideUnlockButtonAction = Runnable { unlockScreenButton.isVisible = false }
 
+    /**
+     * Runnable that hides [playerView] controller
+     */
+    private val hidePlayerViewControllerAction = Runnable { playerView.hideController() }
```

Comment:
```suggestion
    private val hidePlayerViewControllerAction = Runnable {
        playerView.hideController()
    }
```

Reply:
I'd say that's fine, doesn't really need extra wrapping imo. The line above also doesn't do that ^^